specs/_features/eip7732/fork-choice.md

```diff
@@ -255,185 +263,151 @@ def get_checkpoint_block(store: Store, root: Root, epoch: Epoch) -> Root:
-    ):
-        return Gwei(0)
-    committee_weight = get_total_active_balance(state) // SLOTS_PER_EPOCH
-    return (committee_weight * PROPOSER_SCORE_BOOST_EIP7732) // 100
+def should_extend_payload(store: Store, root: Root) -> bool:
+    proposer_root = store.proposer_boost_root
+    return (
+        is_payload_timely(store, root)
+        or proposer_root == Root()
+        or store.blocks[proposer_root].parent_root != root
+        or is_parent_node_full(store, store.blocks[proposer_root])
+    )
 ```
 
-### New `compute_withhold_boost`
-
-This is a similar helper that applies for the withhold boost. In this case this
-always takes into account the reveal status.
+### Modified `get_weight`
 
 ```python
-def compute_withhold_boost(store: Store, state: BeaconState, node: ChildNode) -> Gwei:
-    if store.payload_withhold_boost_root == Root():
-        return Gwei(0)
-    ancestor = get_ancestor(store, store.payload_withhold_boost_root, node.slot)
-    if ancestor.root != node.root:
-        return Gwei(0)
-    if node.slot >= store.blocks[store.payload_withhold_boost_root].slot:
-        ancestor.is_payload_present = store.payload_withhold_boost_full
-    if ancestor.is_payload_present != node.is_payload_present:
-        return Gwei(0)
-
-    committee_weight = get_total_active_balance(state) // SLOTS_PER_EPOCH
-    return (committee_weight * PAYLOAD_WITHHOLD_BOOST) // 100
-```
+def get_weight(store: Store, node: ForkChoiceNode) -> Gwei:
+    if (
+        store.blocks[node.root].slot + 1 == get_current_slot(store)
+        and node.payload_status != PAYLOAD_STATUS_PENDING
+    ):
+        # When deciding on the payload status of a block from
+        # the previous slot, we choose between FULL and EMPTY
+        # based on `should_extend_payload`
+        if node.payload_status == PAYLOAD_STATUS_EMPTY:
+            return Gwei(1)
+        else:
+            # the node has payload status `PAYLOAD_STATUS_FULL`,
+            # and is already known to be locally available
+            return Gwei(2) if should_extend_payload(store, node.root) else Gwei(0)
+    else:
+        state = store.checkpoint_states[store.justified_checkpoint]
+        unslashed_and_active_indices = [
+            i
+            for i in get_active_validator_indices(state, get_current_epoch(state))
+            if not state.validators[i].slashed
+        ]
+        attestation_score = Gwei(
+            sum(
+                state.validators[i].effective_balance
+                for i in unslashed_and_active_indices
+                if (
+                    i in store.latest_messages
+                    and i not in store.equivocating_indices
+                    and is_supporting_vote(store, node, store.latest_messages[i])
+                )
+            )
+        )
 
-### New `compute_reveal_boost`
+        if store.proposer_boost_root == Root():
+            # Return only attestation score if ``proposer_boost_root`` is not set
```

Comment:
We do double backticks in function docstrings (ie `"""..."""`) for mkdocs. Not necessary here.
```suggestion
            # Return only attestation score if `proposer_boost_root` is not set
```